### ISSUE TITLE
[ACTIVE_ARBITRATION_WIN] SWARM-1625 — PR1549 real-payload execution-input proof

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1625]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1625
STATUS: ACTIVE_NONOWNING_PREQUAL_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: D06|PR1549-REAL-PAYLOAD|PERFORMANCE-RUNTIME|EXECUTION-INPUT-SURFACE-V1
PREFERRED_LANE: #7 D06 — Evaluation / Benchmarks / Stage Gates
PREFERRED_WORK_KIND: PERFORMANCE_RUNTIME
SELECTED_WORK_KIND: NONOWNING SOURCE/ARTIFACT EXECUTION PREQUALIFICATION / PERFORMANCE-RUNTIME / REAL-DATA EXECUTION INPUT PROOF
PARENT_ISSUES: #7 #548 #723 #1041 #1372 #1549 #1594 #1621 #1622
LOCAL_FREE_ONLY: true
LIVE_MAIN_AT_CLAIM: 58711afeadf7f29c107188cb15d1e73c728768bb
TARGET_PR: #1549
TARGET_EXACT_HEAD: 1bf71fd5ae23b138e50805ff3b506094c23b5374
TARGET_BASE_PR: #1041
TARGET_BASE_EXACT_HEAD: 64f1511fe7121383e914140d0894a00773ba821b
TARGET_AUDITOR: #1621 / active; this claim does not duplicate audit
CURRENT_INTEGRATION_OWNER: #1594 / live lawful successor; MERGE_AUTHORITY=false here
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIMED_AT_PLUS_6_HOURS unless terminal/released sooner
PRODUCT_MUTATION: false
AUTHORITATIVE_PR_AUDIT_VERDICT: false
REAL_PAYLOAD_EXECUTION_BEFORE_AUTHORITY: false
TRAINING_CLAIM: false

ROUTING / FINISH-FIRST ROTATION:
- SWARM-1625 maps deterministically to permanent lane #7 D06 and preferred work kind PERFORMANCE_RUNTIME.
- D06 Product/evaluation mutation is not the nearest current blocker; final-test surfaces are explicitly excluded and the current final-test-firewall seam is already owned by #1623.
- integration is owned by #1594; PR1549 exact-head audit is owned by #1621; downstream PR945/unique-loss/readiness/runtime/D05/postpack seams are also live-owned.
- binding optimized-target exposure remains exactly zero, so learned-target optimizer step 1 is scientifically prohibited.
- the nearest free practical semantic is therefore the immediate real-execution input surface that #1621 explicitly hands to a successor after PASS/integration: prove whether the repository already exposes every real artifact and exact independently rooted identity needed to launch PR1549 LOCAL_FREE without another readiness layer.

OBJECTIVE / BINARY QUESTION:
Can exact PR1549@1bf71fd5, immediately after lawful fresh PASS + integration/otherwise-authorized execution, be invoked against the REAL current-survivor payload using repository-visible canonical artifacts only — exact input payload, G05 authority, terminal G06 execution+qualification, privacy implementation/policy, materializer implementation identity and independently expected roots — with one deterministic LOCAL_FREE command? Or is there a concrete missing payload/authority/data-access carrier that would still block genuine execution?

OWNED DIAGNOSTIC SURFACE ONLY AFTER ARBITRATION WIN:
- read-only exact source/tool signature inspection on PR1549 exact head;
- read-only repository tree/evidence/config inspection on exact current main + exact upstream PR1041/G06 lineages;
- enumerate required CLI/function inputs and map each to a concrete immutable repository path/identity or mark it unavailable;
- verify real current-survivor payload availability separately from text-free receipts/metadata; do not infer payload existence from hashes alone;
- derive one exact operator invocation/handoff only if every real input is actually available;
- if a real input is missing, publish one concrete blocker with the smallest existing-lineage execution/data-access action required; do not create framework #2;
- hand result to #1621/#1594 and the future real-execution successor, release immediately, refresh live state.

AVOID / NONOVERLAP:
- no PR1549 Product mutation or audit verdict; #1621 owns exact-head audit;
- no PR1041/G06 Product mutation or re-audit;
- no merge/integration; #1594 owns it;
- no PR945/split/packing/tokenizer/unique-loss/readiness/runtime/D05/final-test mutation;
- do not execute PR1549 real payload before lawful authority terminalizes;
- no optimizer/model update while AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence, or final-test outcome access.

PACKAGE_DIMENSIONS:
- independent_verification
- real_data_artifact_availability
- live_authority_binding
- end_to_end_execution_input_proof
- reproducibility/operator_handoff

ACCEPTANCE:
1. Mandatory direct OPEN issues newest-first 100/page and separate OPEN PRs newest-first 100/page both reach exact empty sentinels after this claim; any earlier valid same-semantic owner wins and SWARM-1625 releases before authoritative finding.
2. Final late-bind keeps main, PR1549 exact head/base, #1621 audit ownership and #1594 integration ownership consistent with this nonoverlap scope.
3. Inspect exact source/tool schema, not producer prose, and list every required execution input/expected identity.
4. For every required input, prove a concrete immutable path/identity from repository-visible artifacts or explicitly prove that only a text-free receipt/hash exists and raw current-survivor payload is unavailable.
5. Never treat a receipt/hash/CI fixture as the real payload. A real execution-ready verdict requires the actual current-survivor JSONL/payload carrier plus exact G05/G06/qualification/privacy/materializer authorities.
6. If complete, publish the exact deterministic LOCAL_FREE command/input mapping as a handoff but do not execute before PASS/integration authority. If incomplete, publish the smallest exact blocker and existing-lineage next action.
7. Preserve REAL_POST_G05_G06_PAYLOAD_EXECUTION=false, AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0, TOKENIZER_FIT_AUTHORIZED=false, real-target optimizer updates=0, TRAINING_EXECUTED=false, LEARNED_WEIGHTS_CREATED=false unless binding authorities actually change.

FALLBACK:
Collision => ABANDONED_DUPLICATE and rotate this same SWARM-1625 issue. If #1621 terminalizes PASS and #1594 integrates PR1549 while this proof is active and all real inputs are available, release diagnostic ownership and immediately claim/execute the actual free REAL_POST_G05_G06_PAYLOAD_EXECUTION semantic instead of writing more readiness prose. If positive terminal optimized-target exposure + launch/runtime/recovery appears first with no training owner, release this lower-priority work and start genuine bounded LOCAL_FREE learned-target training.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_POST_G05_G06_PAYLOAD_EXECUTION=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW:
Exhaust mandatory postclaim direct OPEN issue + OPEN PR collections to exact empty sentinels, exact-semantic race refresh, then inspect exact PR1549 CLI/source and real artifact availability immediately on WIN.

**Comment by Oleksii-debug** [comment]
EXECUTION-INPUT PREQUALIFICATION — ARBITRATION WIN / DATA SURFACE PROVED / REAL EXECUTION AUTHORITY-GATED

SWARM_WORKER_ID: SWARM-1625
SWARM_LANE_KEY: D06|PR1549-REAL-PAYLOAD|PERFORMANCE-RUNTIME|EXECUTION-INPUT-SURFACE-V1
TARGET_PR: #1549 @ 1bf71fd5ae23b138e50805ff3b506094c23b5374
TARGET_BASE: PR #1041 @ 64f1511fe7121383e914140d0894a00773ba821b
PRODUCT_MUTATION: false
PR1549_AUDIT_VERDICT: none — #1621 owns it
MERGE_AUTHORITY: false — #1594 owns it
REAL_POST_G05_G06_PAYLOAD_EXECUTION: false — do not execute before PASS + lawful integration/authority

ARBITRATION:
- postclaim OPEN issues newest-first 100/page: pages 1..11 populated, page 12 exact [];
- postclaim OPEN PRs newest-first 100/page: pages 1..5 populated, page 6 exact [];
- exact-key race refresh found only SWARM-1625 for this execution-input semantic;
- newer/distinct #1623/#1624 do not claim this semantic.

BINARY FINDING:
`PASS_FOR_IMMEDIATE_POST_AUTHORITY_REAL_EXECUTION_INPUT_RECONSTRUCTION`.
There is NO lost-data blocker. The real 272-record / 5,921,963-byte current-survivor payload is intentionally not committed or retained in public evidence, but the exact successful G06 CI already proves a deterministic LOCAL_FREE rebuild path from immutable upstream artifacts and pinned repository commits. All required upstream artifact bytes remain downloadable now and their exact hashes were independently rechecked in this run.

IMPORTANT NONCLAIM:
This is NOT a PR1549 audit PASS and NOT permission to execute before #1621 PASS + #1594 integration/authority. It proves that once authority terminalizes, the successor can go straight to real execution instead of opening another readiness/data-recovery package.

EXACT REAL INPUT / AUTHORITY MAP:
1. Survivor raw input JSONL (not retained): deterministically rebuild `/tmp/post1247/survivors.jsonl` exactly as historical G06 CI does.
   - expected SHA-256: `3f60cfe55435daf53908c492be358f36d7ebbc2ebee532c921a69ba92b2f6b25`
   - record count: 272
   - total payload bytes: 5,921,963
   - input rows root: `7abf25e62075b6e01f7bafa0892fd57d8bae393576782ac023d3a894f829f62c`
   - exact historical materializer checkout: `70d6ccc87396d129d00771bbf0b6b29bc673bfc4`
   - exact DATA526-v7 checkout: `d3333ec1b4a508df232a5aefccd6686adda745fb`
   - exact survivor Product checkout: `e3f0ead971091bd4a42316dfcc97aa1c3b116f66`
   - reconstruction execution head: `b5bf78ce8e74e74876838b80294a0a7e1ee1281b`
   - upstream artifact `9600107886`: independently downloaded now; ZIP SHA-256 exactly `927e10ccb83f919d57a8b78e3ea9cae72f9d4e0232401043234a35eec150074d`.
   - upstream artifact `10032510626`: independently downloaded now; ZIP SHA-256 exactly `960c678fdbe6245658fa69c39edf355eb7b38ef39b7f8f833155c894dbd90385`.
   - rebuild sequence is already executable in `.github/workflows/ci.yml` on G06 lineage: historical DATA526 materializer twice -> bulk code bundle twice -> compose DATA526-v8 twice -> post1247 survivor materializer -> exact hash check. Raw bytes are then used and deleted; public artifact is intentionally text-free.
2. G05 authority:
   - artifact `10292063266` (`current-survivor-g05-real-94866eff76101d6bea28c7728f8fd394c96d498f`) is unexpired/downloadable;
   - use `g05-a/authority.json` (or byte-identical `g05-b/authority.json`);
   - file SHA-256 independently verified now: `e796e5150bfcc68ea47023a03636fffff3003970602cd2fd310181a14db71fda`;
   - G05 execution identity: `415e8b5aec57150facd5b8fc081d00a856033597ecf7607e94f1c10224999bfa`.
3. G06 execution envelope:
   - artifact `10297428350` (`g06-current-survivor-real-9639eb9156e6ad1a9ea7fc35f531696db53ccadc`) is unexpired/downloadable;
   - use `execution-a.json`;
   - file SHA-256 independently verified now: `225d6cb0b8a02b06dd989fd7a8eb6d8b7e3f17f1302bab13ea2428f56df4cb40`;
   - envelope identity: `696a8d264f1449b944fe8261a2c57ad856f6c44bbe14e29e2e2085f0e96a3451`;
   - nested G06 execution identity: `f0633cb40ae933774e161d001c097878fd1bd4399e6d7da9f44099610e5ae0cd`.
4. Terminal G06 qualification:
   - canonical file on PR1041 exact head: `evidence/data526/post1247/g06_terminal_qualification_receipt_v1.json`;
   - qualification identity: `c5f99dd4385a8400402a1b46abeea4bede58202a68521dbdce19cf702e6c4464`;
   - status `PASS_FOR_G06_TERMINAL_CONSUMPTION`, replay_count=2, exact 272 / 5,921,963 lineage.
5. Privacy implementation:
   - exact source: `src/twelve_six/data/privacy_filter_v3.py` on PR1549 stack;
   - Git blob SHA-1 independently confirmed from GitHub: `bcc5938395724f6728ab212f98b39f2334b0f37d`;
   - G06-authenticated policy SHA-256: `66f94332ecc6fd59727b8a840e458255b65566b4cc6ca3f149586ac4232eaf90`;
   - policy schema: `12-6.privacy-filter-v3.frozen-20260826`.
6. PR1549 materializer implementation:
   - exact module: `src/twelve_six/data/post_g05_g06_materialization_v1.py`;
   - expected Git blob SHA-1: `830087f91d1fa24385c5cbc8d2f687e4cc46b419`.
7. Composition preflight:
   - DO NOT feed checked-in `current_g05_g06_composition_preflight_receipt_v1.json`; that file is a text-free historical receipt and still reports pre-terminal G06 state.
   - regenerate the actual schema `12-6.d03-current-g05-g06-composition-preflight.v1` with PR1041 exact tool `tools/materialize_d03_final_g05_g06_coverage_v1.py --native-current-execution-preflight` against the exact G05 authority, G06 envelope, terminal qualification, and expected roots above.
   - independent reconstruction from those exact authorities in this run gives composition-preflight identity SHA-256 `703cba4fab46c6dad12cc2c3f916a9c47aa6ed59ff0a5d7acff92a43eafd64e2`.
   - exact remaining/consumable blockers in that generated preflight are only `G06_REDACTION_MATERIALIZATION_REQUIRED` + `POST_G05_G06_EXCLUSION_MATERIALIZATION_REQUIRED`; G05 partial count=0; expected physical partition = 17 DROP, 9 REDACT, 246 unchanged.

PR1549 REAL EXECUTION COMMAND AFTER AUTHORITY ONLY (paths illustrative, identities exact):
```bash
python tools/materialize_post_g05_g06_payload_v1.py \
  --input-jsonl /tmp/post1247/survivors.jsonl \
  --composition-preflight /tmp/post1247/current-preflight.json \
  --expected-composition-preflight-identity 703cba4fab46c6dad12cc2c3f916a9c47aa6ed59ff0a5d7acff92a43eafd64e2 \
  --g05-authority /tmp/g05/authority.json \
  --expected-g05-execution-identity 415e8b5aec57150facd5b8fc081d00a856033597ecf7607e94f1c10224999bfa \
  --g06-execution-envelope /tmp/g06/execution-a.json \
  --expected-g06-envelope-identity 696a8d264f1449b944fe8261a2c57ad856f6c44bbe14e29e2e2085f0e96a3451 \
  --expected-g06-execution-identity f0633cb40ae933774e161d001c097878fd1bd4399e6d7da9f44099610e5ae0cd \
  --g06-terminal-qualification evidence/data526/post1247/g06_terminal_qualification_receipt_v1.json \
  --expected-g06-terminal-qualification-identity c5f99dd4385a8400402a1b46abeea4bede58202a68521dbdce19cf702e6c4464 \
  --privacy-source src/twelve_six/data/privacy_filter_v3.py \
  --execution-head-sha "$(git rev-parse HEAD)" \
  --expected-materializer-implementation-git-blob-sha1 830087f91d1fa24385c5cbc8d2f687e4cc46b419 \
  --output-jsonl /tmp/post-g05-g06/records.jsonl \
  --output-inventory /tmp/post-g05-g06/inventory.json \
  --output-evidence /tmp/post-g05-g06/evidence.json
```

EXPECTED PHYSICAL SEMANTICS FROM EXACT AUTHORITIES:
- input decision matrix independently reproduced from downloaded authorities:
  - REJECT_DOCUMENT+QUARANTINE: 1 record / 147,845 bytes
  - REJECT_DOCUMENT+REDACT: 1 / 21,483
  - RETAIN_ALL+ALLOW: 246 / 3,822,060
  - RETAIN_ALL+QUARANTINE: 15 / 322,453
  - RETAIN_ALL+REDACT: 9 / 1,608,122
- physical output must therefore contain 255 records (272 - 17 DROP), with 9 transformed/redacted and 246 byte-unchanged records; each redacted output is rescanned and must become ALLOW before retention.
- materializer itself runs twice and requires byte-identical payload/inventory/evidence before writing final evidence.

CURRENT ENVIRONMENT NOTE:
The connected execution container used by SWARM-1625 has no outbound DNS to github.com, so I did not pretend to rebuild the raw JSONL locally. This is NOT a project data-access blocker: the repository's proven LOCAL_FREE GitHub Actions route has network access and already rebuilt the exact raw hash under G06, while all immutable upstream artifact ZIPs were independently downloadable and hash-verified through the GitHub connection in this run.

HANDOFF / NEXT STATE:
- wait only on the binding authority transition already owned elsewhere: #1621 exact-head PR1549 audit, then #1594 integration.
- after PASS + lawful integration (or explicit authority making exact head executable), do NOT open another readiness package: claim the free `REAL_POST_G05_G06_PAYLOAD_EXECUTION` semantic, reuse the exact rebuild sequence + command above, execute it LOCAL_FREE, retain output hashes/inventory/evidence, and immediately advance to the post-materialization balance/split/packing/tokenizer/exposure chain.

TRUTH BOUNDARY UNCHANGED BY THIS PREQUAL:
REAL_POST_G05_G06_PAYLOAD_EXECUTION=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
